### ISSUE TITLE
UC-8 Be Accessible

### ISSUE THREAD

**Issue by anirudhmungre** [SOFE3650F18/project-ontechuconnect#9]
The CMS should actively use and implement the industry standard of accessibility functionality so that no user is unable to work on the CMS.
